app/views/helpers/feed/update.phtml

```diff
@@ -8,8 +8,6 @@
 		<a href="<?= _url('stats', 'repartition', 'id', $this->feed->id()) ?>"><?= _i('stats') ?> <?= _t('sub.feed.stats') ?></a>
 	</div>
 
-	<p><?= sanitizeHTML($this->feed->description()) ?></p>
```

Comment:
This was not supposed to be removed. https://github.com/FreshRSS/FreshRSS/issues/5320

Reply:
It was supposed to. See number 4:

> deleted: the feed description. It is editable some centimeters below. It does not make sense to display redundant information.

Reply:
It's not redundant fwiw; the one is plain text input and the removed one is HTML.

Reply:
Here a screen shot of FreshRSS 1.19.0.
What is the benefit to see the description in (1), while it is written in (2) too?

![grafik](https://user-images.githubusercontent.com/1645099/233389469-ca8dfaef-2bf9-4bf2-845b-7e9029d205a5.png)

Reply:
Just for the record here as well: the (1) is in HTML, meaning that it can contain hyperlinks, images, etc.